package.json

```
@@ -73,8 +73,9 @@
       "local-simulator": "quantum/nodes/local-simulator/local-simulator.js",
       "ibm-quantum-system": "quantum/nodes/ibm-quantum-system/ibm-quantum-system.js",
       "circuit-diagram": "quantum/nodes/circuit-diagram/circuit-diagram.js",
+      "script": "quantum/nodes/script/script.js",
+      "grovers-algo":  "quantum/nodes/grovers-algo/grovers-algo.js"
       "bloch-sphere": "quantum/nodes/bloch-sphere/bloch-sphere.js",
```

```suggestion
      "grovers-algo":  "quantum/nodes/grovers-algo/grovers-algo.js",
      "bloch-sphere": "quantum/nodes/bloch-sphere/bloch-sphere.js",
```
Missing comma here is causing the build to fail when it attempts to install dependencies.

---

Thanks!